libs/newsletter-workflow/src/lib/steps/newsletterData/formSchemas.ts

```diff
@@ -40,9 +43,92 @@ export const formSchemas = {
 		})
 		.describe('Choose a theme'),
 
-	description: z
+	signUp: z
 		.object({
+			headline: z.string(),
 			description: z.string(),
 		})
-		.describe('Input a short description to display to users'),
+		.describe('Input the Sign Up page copy'),
+
+	regionFocus: z
+		.object({
+			regionFocus: regionFocusEnumSchema,
+		})
+		.describe('Select from the drop-down list'),
+
+	designBrief: z
+		.object({
+			designBriefDoc: z.string(),
+			figmaDesignUrl: z.string().url().optional(),
+			figmaIncludesThrashers: z.boolean(),
+		})
+		.describe('Input the design brief and Figma design'),
+
+	newsletterHeader: z
+		.object({
+			displayDate: z.boolean(),
+			displayStandfirst: z.boolean(),
+		})
+		.describe('Input the header setup'),
+
+	footer: z
+		.object({
+			email: z.string().email().optional(),
+		})
+		.describe('Input the footer setup'),
+
+	frequency: z
+		.object({
+			frequency: z.string(),
+		})
+		.describe('Input the send frequency'),
+
+	images: z
+		.object({
+			displayImageCaptions: z.boolean(),
+		})
+		.describe('Specify the setup'),
+
+	onlineArticle: z
+		.object({
+			onlineArticle: onlineArticleSchema,
+		})
+		.describe('Select from the drop-down list'),
+
+	linkList: z
+		.object({
+			linkListSubheading: z.string(),
+		})
+		.describe('Input the subheading triggers'),
+
+	podcast: z
+		.object({
+			podcastSubheading: z.string(),
+		})
+		.describe('Input the subheading triggers'),
+
+	readMore: z
+		.object({
+			readMoreSubheading: z.string(),
+			readMoreWording: z.string(),
+			readMoreUrl: z.string().url().optional(),
+		})
+		.describe('Input the Read More setup'),
+
+	tags: z
+		.object({
+			seriesTag: z.string().url().optional(),
+			composerTag: z.string(),
+			composerCampaignTag: z.string(),
+		})
+		.describe('Input the tag setup'),
```

Comment:
I only made seriesTag optional because of the auto validation caused by having it as a url.  You're right that it isn't really a url (so I need to update the example text on that page of the wizard) - I will therefore remove the "optional" for seriesTag - we're not expecting it to exist, but need to know what it should be so that we can automatically email CP to request its creation if it doesn't